app/views/sessions/_login.html.erb

```diff
@@ -0,0 +1,7 @@
+<% if logged_in? %>
+<p class="alert alert-primary">  <%= "Hello,  #{current_user.name}! You're looking mighty fine today!" %>
+  <%= link_to 'Log out', logout_path, method: :delete, class: "alert-link float-right", data: {confirm: 'Are you sure ?'} %>
+<% else %>
+  <%= link_to 'Log in', login_path, class: "alert-link float-right", method: :get %>
+<% end %>
```

Comment:
- Move the if/else statement to a helper method that you call here and render the links and markup within the helper method using a partial